### ISSUE TITLE
Element bug

### ISSUE THREAD

**Issue by ivan-kleshnin** [staltz/cycle#89]
Video proof: https://drive.google.com/file/d/0B9bW0B92XgnAQnVROXNaZ1hxOGM/view

In short:
```javascript
let Model = Cycle.createModel(() => {
  return {
    active$: Rx.Observable.interval(1000).map(() => generateRandomActive(data)),
  }
});

let View = Cycle.createView(Model => ({
  vtree$: Model.get("active$").map(active => {
    return <div><Menu items={data} active={active}/></div>; // this updates every second
    return <Menu items={data} active={active}/>;            // this doesn't ^_^
  }),
}));
```

Playground: https://github.com/ivan-kleshnin/cyclejs-examples/tree/master/examples/4.3-menu-form2

**Comment by staltz** [comment]
Thanks for this. I'll make a test and a fix, seems simple to fix I think.

**Comment by staltz** [comment]
Fixed in [v0.16.2](https://github.com/staltz/cycle/releases/tag/v0.16.2).

**Comment by staltz** [comment]
@ivan-kleshnin I decided to revert on this issue and rather reject the custom element on the root of a View, read release notes for [v0.17.0](https://github.com/staltz/cycle/releases/tag/v0.17.0).
